WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Survey/ProductCreationAISurveyConfirmationView.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+
+/// Hosting controller for `ProductCreationAISurveyConfirmationView`.
+///
+final class ProductCreationAISurveyConfirmationHostingController: UIHostingController<ProductCreationAISurveyConfirmationView> {
+    init(viewModel: ProductCreationAISurveyConfirmationViewModel) {
+        super.init(rootView: ProductCreationAISurveyConfirmationView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Celebration view presented when AI generation is used for the first time
+struct ProductCreationAISurveyConfirmationView: View {
+    private let viewModel: ProductCreationAISurveyConfirmationViewModel
+
+    init(viewModel: ProductCreationAISurveyConfirmationViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack(spacing: Layout.verticalSpacing) {
```

Comment:
Should we make this view scrollable for accessibility font sizes?

Reply:
Done in 725d36d7dfa8aba80c6cbc5cbc5c48955d635e98